pkg/resource/replication_group/testdata/replication_group/cr/rg_cmd_create_initiated.yaml

```diff
@@ -8,6 +8,10 @@ spec:
   replicasPerNodeGroup: 1
   replicationGroupDescription: cluster-mode disabled RG
   replicationGroupID: rg-cmd
+  atRestEncryptionEnabled: false
+  SnapshotRetentionLimit: 0
+  snapshotWindow: "09:00-10:00"
+  TransitEncryptionEnabled: false
```

Comment:
not sure it matters, but `TransitEncryptionEnabled` should be `transitEncryptionEnabled` here and in the similar files below.